### DISCUSSION TITLE
Is there a way to hide a component dynamically?

### DISCUSSION THREAD

**Discussion by Bechir-Brahem** [reflex-dev/reflex#276] [Q&A] [answered]
I want to display a text field only when a state boolean is truthful. 
Is there a pynecone.if() component or a prop such as is_shown to implement this functionality?

**Comment by picklelo** [comment, marked as answer]
You can use `pc.cond`: https://pynecone.io/docs/library/layout/cond

**Reply by Bechir-Brahem** [reply]
Thanks. This is what I've been looking for
